frontends/election-manager/src/app.test.tsx

```diff
@@ -726,6 +729,8 @@ test('tabulating CVRs', async () => {
   );
   const printableArea3 = getByTestId('printable-area');
   expect(within(printableArea3).getAllByText('0').length).toBe(40);
+  // useQuery's in AppRoot are refetching data, and there's no change to wait on
+  await advanceTimersAndPromises();
```

Comment:
Unclear to me what was being fetched and why, but warnings indicated it was one of the hooks in `AppRoot`.